infra/src/main/java/com/redhat/service/smartevents/infra/exceptions/definitions/platform/UnspecifiedProvisioningFailureException.java

```diff
@@ -0,0 +1,12 @@
+package com.redhat.service.smartevents.infra.exceptions.definitions.platform;
+
+public class UnspecifiedProvisioningFailureException extends InternalPlatformException {
```

Comment:
Fallback for any generic `Exception`'s that do not map to explicit `BridgeError`'s.